backend/tests/core/Pims.Core.Test.csproj

```diff
@@ -13,6 +13,10 @@
     <PackageReference Include="Microsoft.EntityFrameworkCore.InMemory" Version="3.1.3" />
     <PackageReference Include="Moq" Version="4.13.1" />
     <PackageReference Include="xunit" Version="2.4.1" />
+    <PackageReference Include="xunit.runner.visualstudio" Version="2.4.1">
```

Comment:
Fixing issue where VS complains about this assembly because it references `xunit`.